docker-run.sh

```diff
@@ -63,6 +66,7 @@ if [[ ! -f /var/www/html/conf/conf.php ]]; then
 \$dolibarr_main_db_pass='${DOLI_DB_PASSWORD}';
 \$dolibarr_main_db_type='${DOLI_DB_TYPE}';
 \$dolibarr_main_authentication='${DOLI_AUTH}';
+\$dolibarr_main_prod=1;
```

Comment:
Can you add a new var DOLI_PROD with default value 1 and update README.template ?

Reply:
Sure